db/schemas/schemas.py

```diff
@@ -36,6 +36,8 @@ class Meta:
     organisation_name = String()
     funding_type = String()
     local_authority = String()
+    cohort = String()
+    is_project_regional = Boolean()
```

Comment:
does this need adding to the returned payload for the frontend (like in the api.yml)?

Reply:
you meant in the components.yml. It is defined as json object there so not required to add https://github.com/communitiesuk/funding-service-design-assessment-store/blob/d69b7677337bb02a666572a98800a7a34dac66f7/openapi/components.yml#L22-L23

Reply:
ah okay, I think that's probably outdated as its missing some schema on the retuned assessment (for example i think 'applicationId' id actually 'application_id')